api/cron/compute-signals.js

```
@@ -415,6 +415,118 @@ function calculateCompositeScore(ta, flow, sentiment) {
+// ─── 패턴 시그널 signal_history 기록 (#373) ────────────────
+// 패턴(SR/DB)만 필터 → KV 쿨다운 NX(하루 1발화/종목) → /api/signal-accuracy POST
+// - signals:latest KV는 건드리지 않음 (CF Worker 소유권 보호, ADR-016)
+// - record_signal_batch RPC가 INSERT-only(dedup 없음)라 크론 쪽에서 선제 차단
+// - priceAtFire = currentPrice(candle 마지막 종가) — 패턴 판정과 동일 소스
+const PATTERN_TYPES = new Set([
+  SIGNAL_TYPES.SUPPORT_RESISTANCE_BREAK,
+  SIGNAL_TYPES.DOUBLE_BOTTOM,
+]);
+const COOLDOWN_TTL = 86400; // 24h — 패턴 horizon(SR 4h, DB 8h)의 상위 바운드
+
+async function recordPatternSignals(signals, req) {
```

ESLint

[error] 'req' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)